test/tests.py

```diff
@@ -1456,6 +1457,34 @@ def test(self):
        ('dml-builtins.dml', None, 'PWUNUSED'),
        ('dml-builtins.dml', None, 'PNO_WUNUSED')]))
 
+
+class DeadMethods(CTestCase):
+    __slots__ = ()
+    def test(self):
+        super().test()
+        c_file = Path(f'{self.cfilename}.c').resolve()
+        dml_sources = dead_dml_methods.dml_sources(c_file)
+        base_file = Path(self.filename).resolve()
+        imported_file = (
+            Path(self.filename).parent / 'dead_methods_imported.dml').resolve()
+        unimported_file = base_file.parent / 'dead_methods_unimported.dml'
+        assert {base_file, imported_file}.issubset(dml_sources), str(
+            dml_sources)
+        (dead, skipped) = dead_dml_methods.find_dead_methods(
+            {c_file}, {base_file, imported_file, unimported_file})
+        assert [p.name for p in skipped] == ['dml-builtins.dml'], f'{skipped}'
+        assert set(dead) == {base_file, imported_file, unimported_file}, dead
+        for (path, lines) in dead.items():
+            blob = path.read_text()
+            expected = [blob[:match.start()].count('\n') + 2
+                             for match in re.finditer('// DEAD', blob)]
+            assert expected == [line for (line, name) in lines], str(
+                (expected, lines))
+
+all_tests.append(DeadMethods(["dead_methods"],
+                             join(testdir, "1.4", "misc", "dead_methods.dml")))
```

Comment:
Add a test to confirm 1.2 methods are ignored entirely?

Reply:
Added.